test/validation.js

```diff
@@ -972,6 +972,16 @@ describe('validation tests', () => {
         .parse()
     })
 
+    it('fails with error if positional argument is unknown', (done) => {
```

Comment:
So this test case fails. With current semantics of `strictCommands` I think this should pass. But making it pass will be a breaking change... Not sure what is the best way to proceed here?